di-ipv-credential-issuer-stub/src/test/java/uk/gov/di/ipv/stub/cred/handlers/AuthorizeHandlerTest.java

```diff
@@ -621,10 +623,14 @@ void apiAuthorizeShouldReturn302WithAuthCodeQueryParamWhenValidAuthRequest()
 
             verify(mockAuthCodeService)
                     .persist(authCoreArgumentCaptor.capture(), anyString(), eq(VALID_REDIRECT_URI));
-            verify(mockContext).redirect(stringArgumentCaptor.capture());
+            verify(mockContext).status(intArgumentCaptor.capture());
```

Comment:
You only need an argument captor if you want to do detailed analysis (like the JSON) - this could just be
```
verify(mockContext).status(200);
```
(This still works though)